File: GO/class3/ex7_discount_value.go
/*Escreva um programa em C que leia o valor de um produto e calcule o
valor final com desconto de 10%. O programa deve imprimir o valor
final na tela.*/

package main

import "fmt"

func main() {

	var productValue, finalProductValue float32

	fmt.Print("Informe o valor de um produto: ")
	fmt.Scan(&productValue)

	finalProductValue = productValue - (productValue * 0.1)

	fmt.Printf("O valor final é: %.2f\n", finalProductValue)
}

File: C/class9/ex46_redoTheQuestions/class3/ex7_discount_value.c
/*Escreva um programa em C que leia o valor de um produto e calcule o
valor final com desconto de 10%. O programa deve imprimir o valor
final na tela.*/

#include <stdio.h>

int main() {

    float productValue;
    float finalProductValue;

    printf("Informe o valor de um produto: ");
    scanf("%f", &productValue);

    finalProductValue = productValue - productValue * (10.0 / 100);
    
    printf("O valor final é: %f\n", finalProductValue);

    return 0;
}